python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1476,6 +1479,9 @@ def plotSkyObjects(self, catalog, description, plotInfoDict, areaDict, zpLabel=N
                                       self.config.analysis, qMin=qMin, qMax=qMax, labeller=AllLabeller(),
                                       ).plotSkyPosition(shortName, plotInfoDict, areaDict,
                                                         dataName="all", **skyplotKwargs)
+        verifyJobFilename = repoInfo.butler.get("coaddAnalysis_verify_job_filename",
+                                                dataId=repoInfo.dataId)[0]
+        self.verifyJob.write(verifyJobFilename)
```

Comment:
These lines need to go above after the `savePlots` call at ~L516 (and, if Arun merges first, it will have already been added there along with some useful metadata: camera, tract, filter, ... 😉 ).

Reply:
Confirmed that Arun has added these lines (and extra metadata) at ~L564, therefore I've deleted these lines from this branch.